src/app/pages/journal/components/journal-navigation/journal-navigation.scss

```diff
@@ -1,6 +1,6 @@
-> div {
-  margin: 12px 0 24px;
-}
+//> div {
```

Comment:
Angular compilation doesn't like ```> div``` syntax, so need to see implications of removing or find resolution.

Reply:
im not a fan of it either. should be simple enough to apply those margins in a better way

Reply:
Removed as syntax incorrect and therefore the output is not evident